src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -1,43 +1,67 @@
 package uk.gov.mca.beacons.service.model;
 
-import java.time.LocalDate;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import java.time.LocalDateTime;
+import java.util.List;
 import java.util.UUID;
 import javax.persistence.Entity;
 import javax.persistence.EntityListeners;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.Transient;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
 import org.springframework.data.annotation.CreatedDate;
-import org.springframework.data.annotation.LastModifiedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 @Entity
 @EntityListeners(AuditingEntityListener.class)
-@Table(name = "beacons")
 public class Beacon {
 
   @Id
   @GeneratedValue
   private UUID id;
 
-  private String beaconType;
+  @Enumerated(EnumType.STRING)
```

Comment:
Annotation ensures that we store the value in the database as the enum string, i.e: `PLB` vs the integer value: `0`